pkg/types/powervs/defaults/platform.go

```diff
@@ -13,4 +17,11 @@ var (
 
```

Comment:
Worth keeping this as a sane default, but If I'm understanding correctly this value will always be overwritten by [this call to SetPlatformDefaults ](https://github.com/openshift/installer/blob/master/pkg/types/defaults/installconfig.go#L124)

Not worth submitting a new revision, but if you wind up fixing anything else here worth mentioning in a comment that this value will be overwritten before deployment.